src/components/OneArticle.jsx

```diff
@@ -0,0 +1,34 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+function OneArticle() {
+  const [singleArticle, setSingleArticle] = useState([]);
+
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    axios
+      .get(
+        `https://backend-service-project-2.onrender.com/api/articles/${article_id}`
+      )
+      .then((response) => {
+        setSingleArticle(response.data.article[0]);
+      });
+  }, [article_id]);

```

Comment:
lovely use of `useParams` 👏 
You _could_ extract the API request to another file and import it in as a function to reduce the number of lines in your React component  🧹